webpack/stories/docs/state-management.stories.mdx

```diff
@@ -0,0 +1,82 @@
+import { Meta } from '@theforeman/stories';
+
+<Meta
+  title="Introduction/Where to Put Stuff"
```

Comment:
Stuff is a good joke, but I'm bit worried people might get confused into thinking it will explain what files to put the code in.
```suggestion
  title="Introduction/Where to Put Data"
```